vm/src/obj/objbool.rs

```diff
@@ -33,6 +33,7 @@ pub fn init(context: &PyContext) {
     let ref bool_type = context.bool_type;
     context.set_attr(&bool_type, "__new__", context.new_rustfunc(bool_new));
     context.set_attr(&bool_type, "__repr__", context.new_rustfunc(bool_repr));
+    context.set_attr(&bool_type, "__doc__", context.new_property(bool_doc));
```

Comment:
Instead of a property and function, use `context.new_str()` to create a python string.